packages/server/src/errors.ts

```diff
@@ -156,24 +88,24 @@ export interface ErrorOptions {
   code?: string;
   // This declaration means it takes any "class" that has a constructor that
   // takes a single string, and should be invoked via the `new` operator.
-  errorClass?: new (message: string) => ApolloError;
+  errorClass?: new (message: string) => GraphQLError;
 }
 
 export function fromGraphQLError(error: GraphQLError, options?: ErrorOptions) {
```

Comment:
Do we need to keep this around? I see we've got a couple call sites but maybe we can simplify those.